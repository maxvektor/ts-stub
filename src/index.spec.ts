(function () {
  'use strict';
  
  test("this is test", () => {
    expect(true).toBe(true);
  });
}());


