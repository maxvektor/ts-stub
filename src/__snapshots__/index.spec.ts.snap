// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adds text to h2 tag 1`] = `"<div><h2>Hello from typescript</h2></div>"`;
